web/src/components/map/utils.js

```diff
@@ -120,3 +128,33 @@ export const canceledCollectionsMessage = canceledRequestMessage("collections");
  * Overload cancel request message with collections
  */
 export const canceledPollutantsMessage = canceledRequestMessage("pollutants");
+
+/**
+ * @param {Object} stylesheet MapBox stylesheet
+ * @returns {Array<Object>} Copies of style layers with ids matching session prefix
+ */
+export const getSessionDataLayerStyles = (stylesheet) => {
+  const sessionStylePrefix = process.env.REACT_APP_MAPBOX_SESSION_STYLE_PREFIX;
```

Comment:
What's the motivation in making this an environment variable, rather than a constant?

Reply:
just so that whoever was managing mapbox wouldn't have to search the code for it, same with the mapbox style, but don't know if it's that big of a difference or if it's worthit, happy to defer!

Reply:
I vote for writing the value of the session_style_prefix directly, rather than using an env variable. We can always refactor if we find we need to access it in different ways, later.